cmd/controller/metrics.go

```diff
@@ -88,6 +88,8 @@ func init() {
 	prometheus.MustRegister(conditionInfo)
 	prometheus.MustRegister(httpRequestsTotal)
 	prometheus.MustRegister(httpRequestDurationSeconds)
+	// Initialise known label values so that counter exists
+	unsealErrorsTotal.WithLabelValues("fetch", "")
```

Comment:
I don't get it... Why should `unsealErrorsTotal` be different from `unsealRequestsTotal`?

Reply:
This PR was ported from this [PR](https://github.com/bitnami-labs/sealed-secrets/pull/612). The explanation about the change are here from the user:

- [Comment 1](https://github.com/bitnami-labs/sealed-secrets/issues/543#issuecomment-883080888)
- [Comment 2](https://github.com/bitnami-labs/sealed-secrets/issues/543#issuecomment-883085565)
- [Comment 3](https://github.com/bitnami-labs/sealed-secrets/issues/543#issuecomment-883086071)

Reply:
code removed [here](https://github.com/bitnami-labs/sealed-secrets/pull/463/files#diff-1b3297fc96400920a6d05a7bba0926014ab0cbf0771f2890286e1d2231b680aaL69)